core/types/transaction_signing.go

```diff
@@ -40,6 +40,8 @@ type sigCache struct {
 func MakeSigner(config *params.ChainConfig, blockNumber *big.Int, blockTime uint64) Signer {
 	var signer Signer
 	switch {
+	case config.IsPrague(blockNumber, blockTime):
+		signer = NewPragueSigner(config.ChainID)
 	case config.IsCancun(blockNumber, blockTime) && !config.IsOptimism():
 		signer = NewCancunSigner(config.ChainID)
```

Comment:
Note that I didn't add the Optimism exception here. I think it's fine for now, but we need to revisit how to disallow blobs when we do L2 Pectra support, cc @danyalprout